build/version.go

```diff
@@ -37,7 +37,7 @@ func BuildTypeString() string {
 }
 
 // BuildVersion is the local build version
-const BuildVersion = "1.17.0-rc3"
+const BuildVersion = "1.17.0-rc4"
```

Comment:
Is this value manually set for every release?

Reply:
Hey @ianconsolata! Yeah, those are set for every release and then `make docsgen` & `make docsgen-cli` is run